grid-cli/docs/vm.md

```diff
@@ -26,6 +26,7 @@ tf-grid-cli deploy vm [flags]
 - memory: memory size in GB (default 1).
 - rootfs: root filesystem size in GB (default 2).
 - ygg: assign yggdrasil ip for VM (default true).
+- gpus: assign alist of gpus to VM. note: setting this without the node option will fail.
```

Comment:
alist -> a list, but i'm not sure how it will look still

Reply:
Should be something like this
```bash
tf-grid-cli deploy vm --name examplevm --ssh ~/.ssh/id_rsa.pub --cpu 2 --memory 4 --disk 10 --gpus gpu1 --gpus gpu2 --node 12```

Reply:
is it the `id` string?

Reply:
can be added to the docs (gpus IDs)?